src/plugins/KartesianProfile.cpp

```diff
@@ -195,21 +258,69 @@ void KartesianProfile::endStep(ParticleContainer *particleContainer, DomainDecom
                 _profiles[i]->reset(uID);
             }
         }
-        accumulatedDatasets = 0;
+        _accumulatedDatasets = 0;
     }
 }
 
-unsigned long KartesianProfile::getUID(ParticleIterator thismol) {
-    auto xun = (unsigned) floor(thismol->r(0) * this->universalInvProfileUnit[0]);
-    auto yun = (unsigned) floor(thismol->r(1) * this->universalInvProfileUnit[1]);
-    auto zun = (unsigned) floor(thismol->r(2) * this->universalInvProfileUnit[2]);
-    auto uID = (unsigned long) (xun * this->universalProfileUnit[1] * this->universalProfileUnit[2]
-                           + yun * this->universalProfileUnit[2] + zun);
+unsigned long KartesianProfile::getUID(ParticleIterator &thismol) {

```

Comment:
After renaming, this should become SpatialProfiles::getCartesianUID(...)

Reply:
changed, e5ff251